run.sh

```diff
@@ -0,0 +1,20 @@
+set -e
+
+PBF_FILE=$1
+
+# launch application
+docker compose down && docker compose up -d
+
+# download sample data
+wget -O data/sample.pbf $PBF_FILE
```

Comment:
```suggestion
wget -O data/sample.pbf ${PBF_FILE}
```

I prefer to have variables like this, because that is easier to see 🤓